snippets/csharp/programming-guide/strings/Strings_1.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace StringCompare
+{
+    class Program
+    {
+        static void Main()
+        {
+            // <SnippetStringInterpolation>
+            var jh = (firstName: "Jupiter", lastName: "Hammon", born: 1711, published: 1761);
+            Console.WriteLine($"{jh.firstName} {jh.lastName} was an African American poet born in {jh.born}.");
+            Console.WriteLine($"He was first published in {jh.published} at the age of {jh.published - jh.born}.");
+            Console.WriteLine($"He'd be over {Math.Round((2018d - jh.born) / 100d) * 100d} years old today.");
+            
+            // Output:
+            // Jupiter Hammon was an African American poet born in 1711.
+            // He was first published in 1761 at the age of 50.
+            // He'd be over 300 years old today. 
+            // <SnippetStringInterpolation>
+            
+            System.Console.WriteLine();
+            
+            // <SnippetStringFormat>
+            var pw = (firstName: "Phillis", lastName: "Wheatley", born: 1753, published: 1773);
+            Console.WriteLine("{0} {1} was an African American poet born in {2}.", pw.firstName, pw.lastName, pw.born);
+            Console.WriteLine("She was first published in {0} at the age of {1}.", pw.published, pw.published - pw.born);
+            Console.WriteLine("She'd be over {0} years old today.", Math.Round((2018d - pw.born) / 100d) * 100d);
+
+            // Output:
+            //Phillis Wheatley was an African American poet born in 1753.
+            //She was first published in 1773 at the age of 20.
+            //She'd be over 300 years old today.
+            // <SnippetStringFormat>
```

Comment:
This snippet tag must be closing.
Also nit: white space in the output after `//` would make it a bit more readable

Reply:
Done and done. This was just sloppy work on my part, sorry about the typos. Also, when I go to push to origin I'll get "hey go fetch", followed by "hey go pull" error messages and "rejected" prompts. I'll fetch and pull and try again. I come back to the PR and there's like six commits sitting there!? Sorry about all the commits. I'm still figuring a lot of this out.

Reply:
@hDoubleE if you finished the work on this PR, please rename it to remove "WIP" and indicate that it's not work in progress anymore.